src/dattobd.c

```diff
@@ -90,17 +90,23 @@ static loff_t noop_llseek(struct file *file, loff_t offset, int origin){
 
 
 #ifndef HAVE_STRUCT_PATH
```

Comment:
`struct path` was introduced in 2.6.20. `struct file` was changed at the same time to use it (vs `struct nameidata` and `d_path`, which were changed in 2.6.25 to use `struct path`).